Dalamud/Game/Gui/ContextMenu/CharacterData.cs

```diff
@@ -0,0 +1,193 @@
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+using Dalamud.Game.ClientState.Resolvers;
+using Dalamud.Memory;
+
+using FFXIVClientStructs.FFXIV.Client.UI.Info;
+
+using Lumina.Excel.GeneratedSheets;
+
+namespace Dalamud.Game.Gui.ContextMenu;
+
+/// <summary>
+/// Dalamud wrapper around a ClientStructs CharacterData.
+/// </summary>
+public unsafe class CharacterData
```

Comment:
Copying from prior thread:

> I’d also suggest moving it somewhere more appropriate as this is for InfoProxy in general and I can see this being useful elsewhere later. This would also likely benefit a rename to `CharacterSearchEntry` or something similar.

Reply:
Yeah, that would work decently. Maybe `Dalamud.Game.Network.Structures.InfoProxy`? We can eventually move Market stuff into its own subdirectory as well.